packages/core/src/state-management/repository.ts

```diff
@@ -487,28 +496,40 @@ export class Repository {
     // We also skip CACAO expiration checking during this initial load as its possible
     // that the CommitID we are being asked to load may in fact be an anchor commit with
     // the timestamp information that will reveal to us that the CACAO didn't actually expire.
-    const base$ = await this.load(commitId.baseID, opts, false)
-
-    return this._atCommit(commitId, base$)
-  }
+    let existingState$ = null
+    try {
+      existingState$ = await this.load(commitId.baseID, opts, false)
+    } catch (err) {
+      this.logger.warn(
+        `Error loading existing state for stream ${commitId.baseID} while loading stream at commit ${commitId.commit}`
+      )
+    }
 
-  private async _atCommit(
-    commitId: CommitID,
-    existingState$: RunningState
-  ): Promise<SnapshotState> {
     return this.executionQ.forStream(commitId).run(async () => {
-      const stateAtCommit = await this.streamLoader.stateAtCommit(existingState$.state, commitId)
+      const stateAtCommit = existingState$
+        ? await this.streamLoader.resetStateToCommit(existingState$.state, commitId)
```

Comment:
this was confusing to follow at first, reading the methods helps. The renaming of stateToCommit to resetStateToCommit helps